### PR TITLE
refactor and add ix.io pastebin

### PR DESCRIPTION
**Pull request by nvinson**

### CODE REVIEW

File: wgetpaste [hunk 1]
```diff
@@ -87,18 +120,19 @@ REGEX_URL_gists='s|^.*"html_url": "\([^"]\+gist[^"]\+\)".*$|\1|p'
...
 escape_input_gists() { sed -e 's|\\|\\\\|g' -e 's|\x1b|\\u001b|g' -e 's|\r||g' -e 's|\t|\\t|g' -e 's|"|\\"|g' -e 's|$|\\n|' <<< "$*" | tr -d '\n'; }
-json_gists() {
-    local description="${1}" language="${2}" content="${3}"
-    [[ "$language" = auto ]] && language="" || language=".$language"
-    echo "{\"description\":\"${description}\",\"public\":\"${PUBLIC_gists}\",\"files\":{\"${description//\/}${language}\":{\"content\":\"${content}\"}}"
+POST_gists() {
+	local description="${2}" language="${3}" content="${6}"
+	[[ "$language" = auto ]] && language="" || language=".$language"
+	echo "{\"description\":\"${description}\",\"public\":\"${PUBLIC_gists}\",\"files\":{\"${description//\/}${language}\":{\"content\":\"${content}\"}}"
+	echo "{\"description\":\"${description}\",\"public\":\"${PUBLIC_gists}\",\"files\":{\"${description//\/}${language}\":{\"content\":\"${content}\"}}"
```
Comment (bcattaneo) [inline wgetpaste:130]: Duplicate `echo`?

**Comment by Cogitri** [comment]
Please squash the changes of the "fix formatting" commit into the commit which actually introduced the tab indentation

**Comment by nvinson** [comment]
Squashed the commit.

**Comment by Cogitri** [comment]
Thanks!

**Pull request merged**